packages/medusa/src/api/routes/admin/product-categories/add-products-batch.ts

```diff
@@ -0,0 +1,116 @@
+import { IsArray, ValidateNested } from "class-validator"
+import { Request, Response } from "express"
+
+import { EntityManager } from "typeorm"
+import { ProductBatchProductCategory } from "../../../../types/product-category"
+import { ProductCategoryService } from "../../../../services"
+import { Type } from "class-transformer"
+import { FindParams } from "../../../../types/common"
+
+/**
+ * @oas [post] /product-categories/{id}/products/batch
+ * operationId: "PostProductCategoriesCategoryProductsBatch"
+ * summary: "Add Products to a category"
+ * description: "Assign a batch of products to a product category."
+ * x-authenticated: true
+ * parameters:
+ *   - (path) id=* {string} The ID of the Product Category.
+ * requestBody:
+ *   content:
+ *     application/json:
+ *       schema:
+ *         $ref: "#/components/schemas/AdminPostProductCategoriesCategoryProductsBatchReq"
+ * x-codegen:
+ *   method: addProducts
+ * x-codeSamples:
+ *   - lang: Shell
+ *     label: cURL
+ *     source: |
```

Comment:
question: should we add medusa-js example here as well?

Reply:
It hasn't been implemented yet, there are separate issues to tackle that separately.